lib/gateway/pipelines.js

```diff
@@ -197,7 +197,8 @@ function validatePipelinePolicies (policies, avaiablePolicies) {
     const policyNames = Object.keys(policyObj);
 
     if (avaiablePolicies.indexOf(policyNames[0]) === -1) {
-      throw new Error(`${policyNames[0]} policy not declared`);
+      log.error(`${policyNames[0]} policy not declared in the 'policies' gateway.config section`);
```

Comment:
Why do You use log here and logger in https://github.com/ExpressGateway/express-gateway/pull/640/files#diff-6dd40befa5e020684f04fdd9e88783b9R44

Reply:
@iwko Because of different name in the top requires. See [here](https://github.com/ExpressGateway/express-gateway/blob/master/lib/gateway/pipelines.js#L3) and [here](https://github.com/ExpressGateway/express-gateway/blob/f04df4711b5400ec36d9b0f4dfe85d4c930654dc/lib/policies/index.js#L3).

Does that clarify?

Reply:
ah, okay I missed that.